browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html

```diff
@@ -150,3 +123,32 @@
     </settings-subpage>
   </template>
 </settings-animated-pages>
+<div hidden="[[isAdBlockRoute_]]">
```

Comment:
q. I'm curious why this was added. Was it removed accidentally?

Reply:
This page is used for two screen.
* brave://settings/shields
* brave://settings/shields/filters

Above second screen is shown when we click `Content filtering` option.
Then,  `isAdBlockRoute_` gives true and we need to hide all except filter subpage(`adblck-subpage` element).